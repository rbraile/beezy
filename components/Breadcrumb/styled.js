import styled from 'styled-components'

export const Menu = styled.nav`
  ul {
    display: flex;
    flex-direction: row;
    padding-left: 10px;
  }

  a {
    color: #094d8c;
  }

  span {
    margin: 0 8px;
  }
`
