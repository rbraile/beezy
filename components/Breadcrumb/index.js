import React from 'react'
import Link from 'next/link'
import { Menu } from './styled'

const Breadcrumb = ({ page }) => {
  return (
    <Menu>
      <ul>
        <li>
          <Link href='/'>
            <a>Home</a>
          </Link>
        </li>
        {page && (
          <li>
            <span>></span>
            <strong>{page}</strong>
          </li>
        )}
      </ul>
    </Menu>
  )
}

export default Breadcrumb
