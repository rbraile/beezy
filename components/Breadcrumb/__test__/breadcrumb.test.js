import React from 'react'
import { mount, shallow } from 'enzyme'

import Breadcrumb from '../index'

describe('TEST = Breadcrumb', () => {
  test('TEST = Breadcrumb = SNAPSHOT', () => {
    const wrapper = mount(<Breadcrumb />)
    expect(wrapper).toMatchSnapshot()
  })
})
