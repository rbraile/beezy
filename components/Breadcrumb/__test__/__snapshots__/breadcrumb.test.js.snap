// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TEST = Breadcrumb TEST = Breadcrumb = SNAPSHOT 1`] = `
<Breadcrumb>
  <styled__Menu>
    <nav
      className="styled__Menu-sc-18gbgtu-0 cxkmwx"
    >
      <ul>
        <li>
          <Link
            href="/"
            prefetch={true}
          >
            <a
              href="/"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              Home
            </a>
          </Link>
        </li>
      </ul>
    </nav>
  </styled__Menu>
</Breadcrumb>
`;
